### PR TITLE
Extensions - Add compileExtension script and update cMake file

### PR DESCRIPTION
**Pull request by PabstMirror**
python script to run 32/64 cmakes (should this be in /tools?)
gets ace version from sqf side
`_x64.dll` rename from ACRE
added `RVExtensionVersion`  

edit:
adds clang-format file
adds `EXPORT void __stdcall RVExtensionVersion(char *output, int outputSize) ` to all exts

### CODE REVIEW

**Comment by jonpas** [comment]
> should this be in /tools?

Yes

File: .github/workflows/extensions.yml [hunk 1]
```diff
@@ -22,7 +22,9 @@ jobs:
     - name: Build
       shell: cmd
       run: |
-        cd extensions/build
+        cd extensions
+        mkdir build
```
Comment (jonpas) [inline .github/workflows/extensions.yml:26]: Why? This folder is Git tracked.

Comment (PabstMirror) [reply .github/workflows/extensions.yml:26]: changed build folders to 
extensions/vcproj32
extensions/vcproj64

File: tools/compileExtensions.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+# vim: set fileencoding=utf-8 :
+
+# make.py
+# An Arma 3 addon build system
+
+###############################################################################
+
+# The MIT License (MIT)
+
+# Copyright (c) 2013-2014 Ryan Schultz
```
Comment (jonpas) [inline tools/compileExtensions.py:11]: ? :D
I don't think this header is correct haha. I added Extension building to make.py anyways, so you can get rid of the entire header.

**Review by jonpas — COMMENTED** [review]
CMakeLists have tabs in various places.

**Pull request closed without merging**